atc/db/resource_check_rate_limiter_test.go

```diff
@@ -32,6 +32,11 @@ var _ = Describe("ResourceCheckRateLimiter", func() {
 		refreshInterval = 5 * time.Minute
 		fakeClock = fakeclock.NewFakeClock(time.Now())
 
+		_, err := psql.Delete("resources").
```

Comment:
guessing this is needed because of the global test suite setup

Reply:
Yes. To cover the test case of `rate.Inf`, have to delete all existing resources. But instead of deleting them, we can also set them inactive.